anvill/src/Lifters/FunctionLifter.cpp

```diff
@@ -115,6 +116,31 @@ static void MuteStateEscape(llvm::CallInst *call) {
   call->setArgOperand(remill::kStatePointerArgNum, undef_val);
 }
 
+static llvm::Function *GetAnvillSwitchFunc(llvm::Module &module,
+                                           bool complete) {
+
+  auto func_name =
+      complete ? kAnvillSwitchCompleteFunc : kAnvillSwitchIncompleteFunc;
+
+  if (auto func = module.getFunction(func_name)) {
```

Comment:
move the auto assignment out, so that the later `auto func = ` can just be `func = `.